app/src/main/res/values/strings.xml

```diff
@@ -1,23 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
-  <string name="app_name">Home Assistant</string>
-  <string name="scanning_network">Scanning the network 
+    <string name="app_name">Home Assistant</string>
+    <string name="scanning_network">Scanning the network
 for Home Assistant</string>
-  <string name="home_assistant_not_discover">Unable to find your 
+    <string name="home_assistant_not_discover">Unable to find your
 Home Assistant instance</string>
-  <string name="connect_to_home_internet">Make sure your phone is connected 
+    <string name="connect_to_home_internet">Make sure your phone is connected
 to your home internet.</string>
-  <string name="scan_again">scan again</string>
-  <string name="manual_setup">enter address manually</string>
-  <string name="input_url">Home Assistant URL</string>
-  <string name="input_url_hint">https://example.duckdns.org:8123</string>
-  <string name="status_of_mobile_app_integration">Status of mobile app integration:</string>
-  <string name="checking_with_home_assistant">Checking with Home Assistant</string>
-  <string name="skip">Skip</string>
-  <string name="retry">Retry</string>
-  <string name="attempting_registration">Attempting to register application…</string>
-  <string name="unable_to_register">Unable to Register Application</string>
-  <string name="error_with_registration">Please check to ensure you have the mobile_app 
+    <string name="scan_again">scan again</string>
+    <string name="manual_setup">enter address manually</string>
+    <string name="input_url">Home Assistant URL</string>
+    <string name="input_url_hint">https://example.duckdns.org:8123</string>
+    <string name="status_of_mobile_app_integration">Status of mobile app integration:</string>
+    <string name="checking_with_home_assistant">Checking with Home Assistant</string>
+    <string name="skip">Skip</string>
+    <string name="retry">Retry</string>
+    <string name="attempting_registration">Attempting to register application…</string>
+    <string name="unable_to_register">Unable to Register Application</string>
+    <string name="error_with_registration">Please check to ensure you have the mobile_app
 integration enabled on your home assistant instance.</string>
-  <string name="url_parse_error">Unable to parse your Home Assistant URL. It should look like https://example.com</string>
+    <string name="url_parse_error">Unable to parse your Home Assistant URL. It should look like https://example.com</string>
+    <string name="pref_location_zone_title">Zone Based Tracking</string>
+    <string name="pref_location_zone_summary">This will take the zones you have in Home Assistant and place geofences around them, updating when you enter or leave.</string>
```

Comment:
More concise comment/description:

Import Home Assistant zones for phone geofence service.

Reply:
I disagree, that sounds robotic. Something like "Import existing Home Assistant zones as geofences for zone based tracking." would be better.

Reply:
I was trying to get it to be short enough for one line. No need for extra space on a setting toggle. I just felt the current one didn't read well.